PocketKit/Sources/PocketKit/Home/CompactHomeCoordinator.swift

```diff
@@ -120,6 +120,10 @@ class CompactHomeCoordinator: NSObject {
         viewModel.$presentedWebReaderURL.sink { [weak self] url in
             self?.present(url: url)
         }.store(in: &slateDetailSubscriptions)
+        
+        viewModel.$sharedActivity.sink { [weak self] activity in
+            self?.present(activity: activity)
+        }.store(in: &subscriptions)
```

Comment:
why not use `slateDetailSubscriptions` ?

Reply:
> Could we do a UI test here?

yup. I'll add that in soon.